### PR TITLE
Use Omni Repo to install Slurm rpms

### PR DESCRIPTION
**Pull request by heitorPB**
**Please provide description of the purpose of this pull request, as well as its
motivation and context.**

Use Omni repo for RPMs

**How was the code tested?**

Please describe the conditions under which the code has been tested.

Depends on https://github.com/omnivector-solutions/slurm-ops-manager/pull/69

Final checklist
- [x] I am the author of these changes, or I have the rights to submit them.
- [x] I added the relevant changed to the README and/or documentation.
- [x] I self reviewed my own code.
- [x] I updated the `CHANGELOG` according to the [contributing
  guide](https://omnivector-solutions.github.io/osd-documentation/master/contributing.html#changelog)

### CODE REVIEW

**Review by jamesbeedy — APPROVED** [review]
Works for me - https://paste.ubuntu.com/p/VvgfQqgpdr/

And I see that `slurm-slurmctld` comes from the omni repo https://paste.ubuntu.com/p/gg6nN3vhqG/

LGTM 👍

File: charm-slurmctld/config.yaml [hunk 1]
```diff
@@ -8,9 +8,9 @@ options:
       This can be set to the Organization's locall mirror/cache of packages and
       superseeds the Omnivector repositories. Alternatively, it can be used to
       track a `testing` Slurm version, e.g. by setting to
-      `ppa:omnivector/osd-testing`.
-
-      Note: currently supported for Ubuntu only.
+      `ppa:omnivector/osd-testing` (on Ubuntu), or
```
Comment (jamesbeedy) [inline charm-slurmctld/config.yaml:11]: We should make a note that this configuration is only applicable post deploy. If the user changes this value after initially deploying the charms then they will have inconsistent versions.

**Review by jamesbeedy — COMMENTED** [review]
What if we set the repo config in `slurmctld` and have slurmctld distribute it to the other charms? In this way we wouldn't be able to end up with a cluster with different slurm version on different nodes.

File: charm-slurmctld/config.yaml (hunk 1 above)
Comment (heitorPB) [reply charm-slurmctld/config.yaml:11]: The advantage of having it in each charm is that it allows the administrator to update/change different daemons. For example: if one wants to update only `slurmdbd`, they can have a specific URL in the `slurmdbd` charm and that would not impact any other charm.

**Review by jamesbeedy — CHANGES_REQUESTED** [review]
Text in config that specifies that the repo config is only honored in pre-deployment configuration.

**Comment by heitorPB** [comment]
> Text in config that specifies that the repo config is only honored in pre-deployment configuration.

@jamesbeedy , this is enough?
https://github.com/omnivector-solutions/osd-documentation/pull/39

**Review by jamesbeedy — APPROVED** [review]
👍

**Pull request merged**